compose/ui/ui/src/skikoTest/kotlin/androidx/compose/ui/node/VoteFrameRateTest.kt

```diff
@@ -0,0 +1,403 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.node
+
+import androidx.compose.ui.FrameRateCategory
+import androidx.compose.ui.graphics.asComposeCanvas
+import androidx.compose.ui.platform.PlatformContext
+import androidx.compose.ui.scene.ComposeSceneInputHandler
+import androidx.compose.ui.scene.PointerEventResult
+import androidx.compose.ui.test.ExperimentalTestApi
+import androidx.compose.ui.test.InternalTestApi
+import androidx.compose.ui.unit.Density
+import androidx.compose.ui.unit.LayoutDirection
+import kotlin.coroutines.CoroutineContext
+import kotlin.coroutines.EmptyCoroutineContext
+import kotlin.test.Test
+import kotlin.test.assertEquals
+import kotlin.test.assertFails
+import kotlin.test.assertTrue
+import kotlinx.coroutines.test.runTest
+import org.jetbrains.skia.Surface
+
+@OptIn(InternalTestApi::class, ExperimentalTestApi::class)
+class VoteFrameRateTest {
+
+    @Test
+    fun testNoVotedFrameRate() = runTest {
+        val surface = Surface.makeRasterN32Premul(100, 100)
+        val owner = RootNodeOwner(
```

Comment:
As alternative: Use scene to test integration with public modifiers API. Since finding nodes/"clicking" is problematic, changes can be done via direct setting state values and re-drawing scene.
Not blocking because of it.